src/models/PostsModel.php

```diff
@@ -16,7 +16,7 @@ public function __construct(PDO $db)
     public function getAll(): array
     {
         $query = $this->db->prepare(
-            'SELECT `posts`.`title`, `posts` . `id`, `posts` . `content`, `posts` . `date_posted` , `posts` . `time_posted`, `users` . `username` 
+    'SELECT `posts`.`title`, `posts` . `id`, `posts` . `content`, `posts` . `date_posted` , `posts` . `time_posted`, `users` . `username`, `posts` . `likes`, `posts` . `dislikes` 
```

Comment:
Formatting: I don't really like these spaces around the `.`s, it doesn't scan well to me.